<html><h3>:x: ci:test - 0 out of 1 jobs passed in 2 minutes 17 seconds 349 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/06458e70062e16a47deb514c1074cc8198db9c30">06458e70062e16a47deb514c1074cc8198db9c30</a></p>0 out of 1 jobs PASSED<div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/7/"><strike><strong>test-subrepository-acceptance-pullrequest(7.1.x)</strong></strike></a></li></ul></div><h5>For more details click <a href="https://test-1-20.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/7/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-20.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/7/">test-subrepository-acceptance-pullrequest(7.1.x)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre><code>prepare-portal-repository:
     [exec] 
     [exec] Warning: Permanently added 'test-1-0,172.16.0.50' (RSA) to the list of known hosts.
     [exec] sent 8 bytes  received 160 bytes  336.00 bytes/sec
     [exec] Warning: Permanently added 'test-1-0,172.16.0.50' (RSA) to the list of known hosts.
     [exec] total size is 0  speedup is 0.00
     [exec] rsync: link_stat "/opt/java/jenkins/userContent/bundles/test-portal-acceptance-upstream(7.1.x)/393/git-hash" failed: No such file or directory (2)
     [exec] rsync: link_stat "/opt/java/jenkins/userContent/bundles/test-portal-acceptance-upstream(7.1.x)/393/git-hash" failed: No such file or directory (2)
     [exec] rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1637) [Receiver=3.1.1]
     [exec] rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1637) [Receiver=3.1.1]
     [exec] rsync: [Receiver] write error: Broken pipe (32)
     [exec] rsync: [Receiver] write error: Broken pipe (32)
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20180831142230585

BUILD FAILED
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-test-subrepository-acceptance-pullrequest.xml:257: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-test-subrepository-acceptance-pullrequest.xml:170: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:695: exec returned: 23</code></pre></div></li></ol></details></html>

--- a/commerce-currency-test/build.gradle
+++ b/commerce-currency-test/build.gradle
@@ -2,4 +2,5 @@ dependencies {
 	testIntegrationCompile group: "com.liferay", name: "com.liferay.arquillian.extension.junit.bridge", version: "1.0.7"
 	testIntegrationCompile group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "3.5.0"
 	testIntegrationCompile project(":apps:commerce:commerce-currency-api")
+	testIntegrationCompile project(":apps:commerce:commerce-currency-test-util")
 }

--- a/commerce-currency-test/src/testIntegration/java/com/liferay/commerce/currency/util/test/CommercePriceFormatterTest.java
+++ b/commerce-currency-test/src/testIntegration/java/com/liferay/commerce/currency/util/test/CommercePriceFormatterTest.java
@@ -16,7 +16,7 @@ package com.liferay.commerce.currency.util.test;
 
 import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
 import com.liferay.commerce.currency.model.CommerceCurrency;
-import com.liferay.commerce.currency.service.CommerceCurrencyLocalService;
+import com.liferay.commerce.currency.test.util.CommerceCurrencyTestUtil;
 import com.liferay.commerce.currency.util.CommercePriceFormatter;
 import com.liferay.portal.kernel.model.Group;
 import com.liferay.portal.kernel.test.rule.AggregateTestRule;
@@ -32,7 +32,6 @@ import java.util.Locale;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -65,31 +64,12 @@ public class CommercePriceFormatterTest {
 		Assert.assertTrue(formattedPrice.matches(regex));
 	}
 
-	@Ignore
-	@Test
-	public void testFormatCurrencyBigDecimalDefaultFormatter()
-		throws Exception {
-
-		String regex = "^[" + _SYMBOLS + "]\\d{1,3}(\\,\\d{3})*\\.\\d\\d$";
-
-		CommerceCurrency commerceCurrency =
-			_commerceCurrencyLocalService.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
-
-		String formattedPrice = _commercePriceFormatter.format(
-			commerceCurrency, _price, Locale.US);
-
-		Assert.assertTrue(formattedPrice.matches(regex));
-	}
-
-	@Ignore
 	@Test
 	public void testFormatCurrencyBigDecimalLocaleFR() throws Exception {
 		String regexFR = "^\\d{1,3}(.\\d{3})*,\\d\\d\\s[" + _SYMBOLS + "]$";
 
 		CommerceCurrency commerceCurrency =
-			_commerceCurrencyLocalService.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		commerceCurrency.setFormatPattern("###,##0.00 $", Locale.FRANCE);
 
@@ -101,14 +81,12 @@ public class CommercePriceFormatterTest {
 		Assert.assertTrue(formattedPrice.matches(regexFR));
 	}
 
-	@Ignore
 	@Test
 	public void testFormatCurrencyBigDecimalLocaleIT() throws Exception {
 		String regexIT = "^[" + _SYMBOLS + "]\\s\\d{1,3}(\\.\\d{3})*,\\d\\d$";
 
 		CommerceCurrency commerceCurrency =
-			_commerceCurrencyLocalService.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		commerceCurrency.setFormatPattern("$ ###,##0.00", Locale.ITALY);
 
@@ -120,14 +98,12 @@ public class CommercePriceFormatterTest {
 		Assert.assertTrue(formattedPrice.matches(regexIT));
 	}
 
-	@Ignore
 	@Test
 	public void testFormatCurrencyBigDecimalLocaleUS() throws Exception {
 		String regexUS = "^[" + _SYMBOLS + "]\\d{1,3}(\\,\\d{3})*\\.\\d\\d$";
 
 		CommerceCurrency commerceCurrency =
-			_commerceCurrencyLocalService.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		commerceCurrency.setFormatPattern("$###,##0.00", Locale.US);
 
@@ -140,9 +116,6 @@ public class CommercePriceFormatterTest {
 	}
 
 	private static final String _SYMBOLS = "€$¥£R$₹";
-
-	@Inject
-	private CommerceCurrencyLocalService _commerceCurrencyLocalService;
 
 	@Inject
 	private CommercePriceFormatter _commercePriceFormatter;

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/discount/test/CommerceDiscountTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/discount/test/CommerceDiscountTest.java
@@ -48,7 +48,6 @@ import org.frutilla.FrutillaRule;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -56,7 +55,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class CommerceDiscountTest {
 

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/discount/test/CommerceOrderDiscountTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/discount/test/CommerceOrderDiscountTest.java
@@ -47,7 +47,6 @@ import java.math.BigDecimal;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -55,7 +54,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class CommerceOrderDiscountTest {
 

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/order/test/CommerceOrderHttpHelperImplTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/order/test/CommerceOrderHttpHelperImplTest.java
@@ -30,6 +30,7 @@ import com.liferay.commerce.test.util.TestCommerceContext;
 import com.liferay.petra.lang.CentralizedThreadLocal;
 import com.liferay.portal.kernel.model.Group;
 import com.liferay.portal.kernel.model.User;
+import com.liferay.portal.kernel.security.auth.PrincipalThreadLocal;
 import com.liferay.portal.kernel.security.permission.PermissionCheckerFactoryUtil;
 import com.liferay.portal.kernel.test.rule.AggregateTestRule;
 import com.liferay.portal.kernel.test.rule.DeleteAfterTestRun;
@@ -50,7 +51,6 @@ import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -60,7 +60,6 @@ import org.springframework.mock.web.MockHttpServletRequest;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class CommerceOrderHttpHelperImplTest {
 
@@ -76,6 +75,8 @@ public class CommerceOrderHttpHelperImplTest {
 		_group = GroupTestUtil.addGroup();
 
 		_user = UserTestUtil.addUser();
+
+		PrincipalThreadLocal.setName(_user.getUserId());
 
 		_httpServletRequest = new MockHttpServletRequest();
 

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/search/test/CommerceOrderItemIndexerTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/search/test/CommerceOrderItemIndexerTest.java
@@ -16,7 +16,7 @@ package com.liferay.commerce.internal.search.test;
 
 import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
 import com.liferay.commerce.currency.model.CommerceCurrency;
-import com.liferay.commerce.currency.service.CommerceCurrencyLocalServiceUtil;
+import com.liferay.commerce.currency.test.util.CommerceCurrencyTestUtil;
 import com.liferay.commerce.internal.search.CommerceOrderItemIndexer;
 import com.liferay.commerce.internal.test.util.CommerceTestUtil;
 import com.liferay.commerce.model.CommerceOrder;
@@ -51,7 +51,6 @@ import java.util.List;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -59,7 +58,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Andrea Di Giorgi
  */
-@Ignore
 @RunWith(Arquillian.class)
 @Sync
 public class CommerceOrderItemIndexerTest {
@@ -94,8 +92,7 @@ public class CommerceOrderItemIndexerTest {
 		CommerceOrderItem[] commerceOrderItems = new CommerceOrderItem[count];
 
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/util/test/CommerceShippingHelperTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/internal/util/test/CommerceShippingHelperTest.java
@@ -16,7 +16,7 @@ package com.liferay.commerce.internal.util.test;
 
 import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
 import com.liferay.commerce.currency.model.CommerceCurrency;
-import com.liferay.commerce.currency.service.CommerceCurrencyLocalServiceUtil;
+import com.liferay.commerce.currency.test.util.CommerceCurrencyTestUtil;
 import com.liferay.commerce.internal.test.util.CommerceTestUtil;
 import com.liferay.commerce.model.CommerceOrder;
 import com.liferay.commerce.model.CommerceWarehouse;
@@ -42,7 +42,6 @@ import java.math.BigDecimal;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -50,7 +49,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class CommerceShippingHelperTest {
 
@@ -72,8 +70,7 @@ public class CommerceShippingHelperTest {
 	@Test
 	public void testGetDimensions() throws Exception {
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());
@@ -129,8 +126,7 @@ public class CommerceShippingHelperTest {
 	@Test
 	public void testGetWeight() throws Exception {
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/order/stock/test/OrderStockManagementTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/order/stock/test/OrderStockManagementTest.java
@@ -16,7 +16,7 @@ package com.liferay.commerce.order.stock.test;
 
 import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
 import com.liferay.commerce.currency.model.CommerceCurrency;
-import com.liferay.commerce.currency.service.CommerceCurrencyLocalServiceUtil;
+import com.liferay.commerce.currency.test.util.CommerceCurrencyTestUtil;
 import com.liferay.commerce.exception.CommerceOrderValidatorException;
 import com.liferay.commerce.internal.test.util.CommerceTestUtil;
 import com.liferay.commerce.model.CommerceOrder;
@@ -42,7 +42,6 @@ import org.frutilla.FrutillaRule;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -50,7 +49,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class OrderStockManagementTest {
 
@@ -79,8 +77,7 @@ public class OrderStockManagementTest {
 			"The product will be added even if no stock is available"
 		);
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());
@@ -118,8 +115,7 @@ public class OrderStockManagementTest {
 			"The product's stock availability is updated"
 		);
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());
@@ -178,8 +174,7 @@ public class OrderStockManagementTest {
 			"The product will be successfully added to the order"
 		);
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());
@@ -216,8 +211,7 @@ public class OrderStockManagementTest {
 			"The action will fail due to stock unavailability"
 		);
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());
@@ -245,8 +239,7 @@ public class OrderStockManagementTest {
 			"The action will fail due to stock unavailability"
 		);
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());
@@ -280,8 +273,7 @@ public class OrderStockManagementTest {
 			"The action will fail due to stock unavailability"
 		);
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder1 = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/organization/service/impl/test/CommerceOrganizationLocalServiceImplTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/organization/service/impl/test/CommerceOrganizationLocalServiceImplTest.java
@@ -37,7 +37,6 @@ import org.junit.Assert;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -47,7 +46,6 @@ import org.osgi.service.cm.ConfigurationAdmin;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class CommerceOrganizationLocalServiceImplTest {
 

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/service/test/CommerceOrderItemLocalServiceTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/service/test/CommerceOrderItemLocalServiceTest.java
@@ -52,7 +52,6 @@ import org.frutilla.FrutillaRule;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -60,7 +59,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class CommerceOrderItemLocalServiceTest {
 

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/service/test/CommerceWarehouseFinderTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/service/test/CommerceWarehouseFinderTest.java
@@ -35,7 +35,6 @@ import java.util.List;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -43,7 +42,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Andrea Di Giorgi
  */
-@Ignore
 @RunWith(Arquillian.class)
 @Sync
 public class CommerceWarehouseFinderTest {

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/test/CommerceCheckoutTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/test/CommerceCheckoutTest.java
@@ -16,7 +16,7 @@ package com.liferay.commerce.test;
 
 import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
 import com.liferay.commerce.currency.model.CommerceCurrency;
-import com.liferay.commerce.currency.service.CommerceCurrencyLocalServiceUtil;
+import com.liferay.commerce.currency.test.util.CommerceCurrencyTestUtil;
 import com.liferay.commerce.internal.test.util.CommerceTestUtil;
 import com.liferay.commerce.model.CommerceOrder;
 import com.liferay.commerce.model.CommerceOrderItem;
@@ -38,7 +38,6 @@ import java.util.List;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -46,7 +45,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class CommerceCheckoutTest {
 
@@ -63,8 +61,7 @@ public class CommerceCheckoutTest {
 	@Test
 	public void testUserCheckout() throws Exception {
 		CommerceCurrency commerceCurrency =
-			CommerceCurrencyLocalServiceUtil.fetchPrimaryCommerceCurrency(
-				_group.getGroupId());
+			CommerceCurrencyTestUtil.addCommerceCurrency(_group.getGroupId());
 
 		CommerceOrder commerceOrder = CommerceTestUtil.addUserCommerceOrder(
 			_group.getGroupId(), 0, commerceCurrency.getCommerceCurrencyId());

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/test/CommercePriceListUserSegmentTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/test/CommercePriceListUserSegmentTest.java
@@ -44,7 +44,6 @@ import org.frutilla.FrutillaRule;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -52,7 +51,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class CommercePriceListUserSegmentTest {
 

--- a/commerce-test/src/testIntegration/java/com/liferay/commerce/test/CommercePriceListUserSegmentWithOrganizationTest.java
+++ b/commerce-test/src/testIntegration/java/com/liferay/commerce/test/CommercePriceListUserSegmentWithOrganizationTest.java
@@ -44,7 +44,6 @@ import org.frutilla.FrutillaRule;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
@@ -52,7 +51,6 @@ import org.junit.runner.RunWith;
 /**
  * @author Luca Pellizzon
  */
-@Ignore
 @RunWith(Arquillian.class)
 public class CommercePriceListUserSegmentWithOrganizationTest {
 

